src/elements/JobInternalArtifacts/JobInternalArtifacts.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+import prettyBytes from 'pretty-bytes'
+import { connect } from 'react-redux'
+
+import jobsActions from '../../actions/jobs'
+import { formatDatetime } from '../../utils'
+
+import downloadIcon from '../../images/download-icon.png'
+
+const JobInternalArtifacts = ({ jobsStore, getArtifacts }) => {
+  const items = jobsStore.selectedJob.artifacts.map(item => {
+    const index = item.target_path.indexOf('://')
+    const target_path = {
+      schema: item.target_path.includes('://')
+        ? item.target_path.slice(0, index)
+        : null,
+      path: item.target_path.includes('://')
+        ? item.target_path.slice(index + '://'.length)
+        : item.target_path
+    }
+    return {
+      key: item.key,
+      target_path: target_path,
+      size: prettyBytes(item.size),
+      date: formatDatetime(jobsStore.selectedJob.startTime)
+    }
+  })
+
+  const handleClick = (e, schema, path) => {
+    e.persist()
+    getArtifacts(schema, path).then(() => {
+      e.target
+        .closest('.jobs__table__item_artifacts__table_item')
+        .classList.add('view')
+    })
+  }
+
+  return (
+    <div className="jobs__table__item_artifacts">
+      <table>
+        <tbody>
+          {items.map((item, i) => {
+            return [
+              <tr key={i} className="jobs__table__item_artifacts__table_item">
+                <td
+                  onClick={e =>
+                    handleClick(
+                      e,
+                      item.target_path.schema,
+                      item.target_path.path
+                    )
+                  }
+                >
+                  {item.key}
+                </td>
+                <td>
+                  {item.target_path.schema && (
+                    <p>schema: {item.target_path.schema}</p>
+                  )}
+                  path: {item.target_path.path}
+                </td>
+                <td>size: {item.size}</td>
+                <td>{item.date}</td>
+                <td>
+                  <button>
+                    <Link
+                      to="/"
+                      target="_blank"
+                      download={`/files?${
+                        item.target_path.schema
+                          ? `schema=${item.target_path.schema}`
+                          : ''
+                      }&path=${item.target_path.path}`}
```

Comment:
This is a bug.
If schema is undefined, the download link is `/files?&path=<some_path>`
The `?&` is wrong syntax.
I suggest to swap `path` and `schema`. `path` always exist, `schema` does not _always_ exist. The `&` needs to be there only if both `path` and `schema` are there.